Agreements/External/Rat_SNE.txt

```diff
@@ -1,5 +1,40 @@
-FIR_COPX:EGPH:*:NIMAT:*:*:Rathlin:EISN 4AU:31000:*:NIMAT
-FIR_COPX:EGPF:*:NIMAT:*:*:Rathlin:EISN 4AU:31000:*:NIMAT
+;Specific Transfer Agreements
 FIR_COPX:EGPK:*:NIMAT:*:*:Rathlin:EISN 4AU:31000:*:NIMAT
 
 FIR_COPX:*:*:ERNAN:EIDW:*:Rathlin:EISN 4AU:*:29000:ERNAN
+
+;All COPs
+FIR_COPX:*:*:IBROD:*:*:Rathlin:EISN 1AS:*:*:IBROD
```

Comment:
GOMUP?

Reply:
GOMUP is in the ScAC Central sector, and is a COP with Shanwick, so nothing required here